invenio_access/permissions.py

```diff
@@ -8,54 +8,55 @@
 
 """Permission and action needs for Invenio."""
 
-import warnings
 from collections import namedtuple
 from functools import partial
 from itertools import chain
 
 from flask_principal import ActionNeed, Need
 from flask_principal import Permission as _Permission
 
-from .models import ActionRoles, ActionSystemRoles, ActionUsers, \
-    get_action_cache_key
+from .models import (
+    ActionRoles,
+    ActionSystemRoles,
+    ActionUsers,
+    get_action_cache_key,
+)
 from .proxies import current_access
 
-_Need = namedtuple('Need', ['method', 'value', 'argument'])
+_Need = namedtuple("Need", ["method", "value", "argument"])
```

Comment:
Question: Is this a convention? single vs. double quotes?

Reply:
Nope, I am using `Black` on "new" code, and it goes for `"`

Reply:
Actually, sorry for the "noise" in the PR